src/views/LimitedRestorationExtension.vue

```diff
@@ -305,6 +309,10 @@ export default class LimitedRestorationExtension extends Vue {
     return this.getResource.changeData?.orgPersonInfo.orgPersonLabel
   }
 
+  get userEmailOptional (): boolean {
+    return this.getResource.userEmailOptional
+  }
+
   // build applicant orgPerson and assign id (uuid)
```

Comment:
If you use a JSDoc style comment then your IDE tools should show you the comments when you hover on the function.

Reply:
Ditto for LimitedRestorationToFull.vue line 316.

Reply:
Thanks. LGTM.